rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxViewGroup.java

```diff
@@ -0,0 +1,21 @@
+package com.jakewharton.rxbinding.view;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.ViewGroup;
+import rx.Observable;
+
+public final class RxViewGroup {
+  /**
+   * Create an observable of hierarchy change events for {@code viewGroup}.
```

Comment:
This Javadoc does not match the other operators which include a warning and details of the use of a setter.